client/my-sites/plugins/plugin-details.jsx

```diff
@@ -94,6 +95,10 @@ function PluginDetails( props ) {
 	const isWporgPluginFetched = useSelector( ( state ) =>
 		isWporgPluginFetchedSelector( state, props.pluginSlug )
 	);
+	const wporgPluginError = useSelector( ( state ) =>
+		isWporgPluginErrorSelector( state, props.pluginSlug )
+	);
+	const wporgPluginNotFound = wporgPluginError?.response?.status === 404;
```

Comment:
Minor question, what do you think about adding that logic to the selector and renaming it to something like `isPluginFound` or something around that line? Just to avoid having this logic in the component.

Reply:
We have a similar selector called `isWporgPluginFetchedSelector` which can be confusing with `isPluginPluginFoundSelector`. And this selector cannot be used here because it tells nothing about the status code returned.